servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -292,11 +292,55 @@ public String toString() {
     return sb.toString();
   }
 
+  /**
+   * Converts the entry to an {@code InodeFile}.
+   *
+   * @return the {@code InodeFile} representation
+   */
+  public static InodeFile fromJournalEntry(InodeFileEntry entry) {
+    InodeFile inode =
+        new InodeFile.Builder()
+            .setName(entry.getName())
+            .setBlockContainerId(BlockId.getContainerId(entry.getId()))
+            .setBlockSizeBytes(entry.getBlockSizeBytes())
+            .setCacheable(entry.getCacheable())
+            .setCreationTimeMs(entry.getCreationTimeMs())
+            .setLastModificationTimeMs(entry.getLastModificationTimeMs())
+            .setParentId(entry.getParentId())
+            .setPersisted(entry.getPersisted())
+            .setPinned(entry.getPinned())
+            .setTTL(entry.getTtl())
```

Comment:
Since Proto (and Thrift) use getTtl/setTtl, our code should probably do that too. I agree with @aaudiber that it makes sense to do that in a follow up though.